bubble_sort/lib/bubble_sort_by.rb

```diff
@@ -0,0 +1,15 @@
+def bubble_sort_by(array)
```

Comment:
The function is working as it should but it's returning an error when the block is not provided. Kindly fix it so it should work without block as well. You can always use a spaceship operator for comparison as well.

Reply:
@inhaq Fixed 👍